pallets/asset/src/lib.rs

```diff
@@ -920,7 +922,11 @@ decl_error! {
         /// Registration of ticker has expired.
         TickerRegistrationExpired,
         /// Transfers to self are not allowed
-        SenderSameAsReceiver
+        SenderSameAsReceiver,
+        /// After registering the confidential asset, its total supply can change once from zero to a positive value.
+        CanSetTotalSupplyOnlyOnce,
+        /// After registering the confidential asset, its total supply can change once from zero to a positive value.
+        TotalSupplyMustBePositive,
```

Comment:
I suppose we can remove them from the asset pallet as it gets moved in confidential-asset pallet.

Reply:
Right, I had moved them to confidential asset pallet. Seems like during the rebasing, these were left here by mistake. I'll remove them in the next commit and merge the PR :)